src/helpers/applyProps.ts

```diff
@@ -85,7 +85,7 @@ export function applyProps(
         let currentInstance = instance;
         let targetProp = currentInstance[key];
 
-        if ((key === 'draw') && (typeof value === 'function'))
+        if ((key as string === 'draw') && (typeof value === 'function'))
```

Comment:
Not my favorite hack, after refactoring the draw callback prop on the types, but it does work. 🤣